pytorch_lightning/plugins/training_type/parallel.py

```diff
@@ -47,6 +47,10 @@ def root_device(self):
     def on_gpu(self):
         return self.root_device.type == "cuda" and torch.cuda.is_available()
 
+    @property
+    def use_spawn(self) -> bool:
+        return False
```

Comment:
I would avoid this. Can we do isinstance check?

Reply:
Yes, but I find it less clean.

Reply:
The problem is this style of adding properties that identify each plugin will not scale long term and at the finest granularity it is the same as an instance check. 

it could be part of the accelerator connector refactor one day. The use_spawn could be handled by the connector.